### PR TITLE
Mobile portfolio body

### PR DESCRIPTION
**Pull request by Kamba56**
Added work sections
Added about me section
Added contact section
Styled the sections
Added screenshots to README.md

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,22 +15,280 @@
...
+              <ul class="language flex-row">
+                <li>Ruby on rails</li>
+                <li>Css</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+
+              <button type="button" class="btn">See Project</button>
+            </div>
+          </div>
```
Comment (ahmetbozaci) [inline index.html]: - [ ] The original design has 6 project cards. Please delete last one

File: index.html [hunk 2]
```diff
@@ -15,22 +15,280 @@
...
+          <input type="email" name="email" placeholder="Email address" />
+          <textarea name="comment" id="comment" cols="30" rows="10">
+Enter text here</textarea
+          >
+          <button type="submit" class="btn">Get in touch</button>
+        </form>
+      </section>
+    </main>
+
+    <footer class="flex-column container center-align">
```
Comment (ahmetbozaci) [inline index.html]: - [ ] Nice to add a contact section but this is not in this milestone. Please add it to the next milestone in another branch, so another TSE can review it. For that, you can just comment on this contact section.

- Comment all part after skills section.

image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @Kamba56  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._
- [ ] Please add a background image in the about me section like in the original
image
YOUR DESIGN:
image


___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏

almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: README.md [hunk 3]
```diff
@@ -14,9 +19,9 @@
...
 👤 **Toma John Musa**
 
-- GitHub: [@githubhandle](https://github.com/Kamba56)
-- Twitter: [@twitterhandle](https://twitter.com/Kamba_TJ)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/toma-john-47092622b)
+- GitHub: [@Kamba56](https://github.com/Kamba56)
+- Twitter: [@Kamba_TJ](https://twitter.com/Kamba_TJ)
+- LinkedIn: [Toma](https://linkedin.com/in/toma-john-47092622b)
 
 ## Show your support
```
Comment (medaminedev66) [inline README.md:27]: [OPTIONAL] The MIT link is broken. Kindly fix it. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.

**Review by medaminedev66 — APPROVED** [review]
## Hi @Kamba56, 

# Status: Approved ✔️
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (Kamba56) [reply README.md:27]: @medaminedev66 Please I don't know what link you are talking about, i checked and everything works fine.

**Pull request merged**